lib/aggregate.js

```diff
@@ -12,7 +12,11 @@ var elmTestGeneratedDir = path.join(
     "0.19.1-revision2"
 );
 
-module.exports = function(sourcePath) {
+var revertOutOfRootFix = (path) => {
+  return path.replace(/__dotdot/g,'..');
+}
```

Comment:
This is the part where we try to translate from `__dotdot` back to `..`. We can't do it too quick because of the `path.split('.')` inside the thing that later tries to translate `Foo.Bar` back to `Foo/Bar.elm`